remoteworkitem/github_test.go

```diff
@@ -0,0 +1,37 @@
+package remoteworkitem
+
+import (
+	"testing"
+
+	"github.com/almighty/almighty-core/resource"
+	"github.com/google/go-github/github"
+)
+
+type FakeGithubIssueFetcher struct{}
+
+// ListIssues list all issues
+func (f *FakeGithubIssueFetcher) ListIssues(query string, opts *github.SearchOptions) (*github.IssuesSearchResult, *github.Response, error) {
+	if opts.ListOptions.Page == 0 {
+		one := 1
+		i := github.Issue{ID: &one}
+		isr := &github.IssuesSearchResult{Issues: []github.Issue{i}}
+		r := &github.Response{}
+		r.NextPage = 1
+		return isr, r, nil
+	}
+	isr := &github.IssuesSearchResult{}
+	r := &github.Response{}
+	r.NextPage = 0
+	return isr, r, nil
+
+}
+func TestGithubFetch(t *testing.T) {
+	resource.Require(t, resource.Database)
```

Comment:
@kwk yeah. DB is not required.

Thanks for all the help @kwk & @tsmaeder for teaching me mock testing!